lstchain/scripts/lstchain_data_r0_to_dl1.py

```diff
@@ -159,30 +140,22 @@ def main():
         config['source_config']['EventSource']['max_events'] = args.max_events
 
     lst_event_source = config['source_config']['LSTEventSource']
-    if args.ucts_t0_dragon is not None:
-        lst_event_source['EventTimeCalculator']['ucts_t0_dragon'] = \
-            args.ucts_t0_dragon
-    if args.dragon_counter0 is not None:
-        lst_event_source['EventTimeCalculator']['dragon_counter0'] = \
-            args.dragon_counter0
-    if args.ucts_t0_tib is not None:
-        lst_event_source['EventTimeCalculator']['ucts_t0_tib'] = \
-            args.ucts_t0_tib
-    if args.tib_counter0 is not None:
-        lst_event_source['EventTimeCalculator']['tib_counter0'] = \
-            args.tib_counter0
+    time_calculator = lst_event_source['EventTimeCalculator']
```

Comment:
Won't this set the reference values to null if they are not provided in the args? (overriding possibly valid values set through the json file)